tools/template/cartesi-machine.template

```diff
@@ -2,4 +2,4 @@
 export LUA_PATH="$(lua5.3 -e 'print(package.path)');ARG_LUA_PATH"
```

Comment:
An option to completely avoid this problem:

If we install things correctly in `/usr`, this proxy script is unnecessary, we could simplify and install `/usr/bin/cartesi-machine.lua` directly as `/usr/bin/cartesi-machine`. The images path should also be auto detectable. I think this proxy script is more relevant when we are developing rather than distributing.

Reply:
Should be fixed on the fixups.